code-challenge/src/components/Search/Search.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import './Search.scss';
+import {IconButton, InputBase, Paper} from "@material-ui/core";
+import SearchIcon from "@material-ui/icons/Search";
+import {useDispatch} from "react-redux";
+
+const Search = () => {
+    const dispatch = useDispatch();
+
+    const updateArticles = (event: any) => {
+        if (event.target.value || event.target.value === '') {
+            dispatch({ type: 'FILTER_ARTICLES', payload: { searchTerm: event.target.value } });
+        }
+    }
+
+    return (
+        <Paper className="Search" component="form">
+            <InputBase
+                className="SearchInputBase"
+                placeholder="Search by title, byline, or section"
+                inputProps={{ 'aria-label': 'Search by title, byline, or section' }}
```

Comment:
I would have thought that material UI would be more proactive in adding aria label based on the provided placeholder. It seems strange that you have to explicitly include accessibility and it doesn't come pre-configured out of the box.

Reply:
but that would be far too easy